packages/drivers/odsp-driver-definitions/package.json

```diff
@@ -53,16 +53,6 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {
-			"EnumDeclaration_OdspErrorType": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_IOdspError": {
-				"backCompat": false
-			},
-			"TypeAliasDeclaration_OdspError": {
-				"backCompat": false
-			}
-		}
+		"broken": {}
```

Comment:
Next instance of 3: Additional values added to 'OdspErrorType' enum no longer flagged as backCompat issue.